### PR TITLE
History: clean search bar inline button

### PR DESCRIPTION
**Pull request by rating89us**
image

### CODE REVIEW

File: pages/History.qml [hunk 1]
```diff
@@ -170,6 +170,21 @@ Rectangle {
                         root.updateFilter();
                     }
                 }
+
+                MoneroComponents.InlineButton {
+                    Layout.topMargin: -8
+                    Layout.rightMargin: -8
+                    Layout.leftMargin: 8
```
Comment (selsta) [inline pages/History.qml:177]:
```suggestion
                    Layout.leftMargin: -2
```

Comment (rating89us) [reply pages/History.qml:177]: I think it's necessary enough blank space after the last character, so the user can click at the end of the search bar content with the mouse. By using a small left margin it's easy to accidentally click the clean search button.

Comment (selsta) [reply pages/History.qml:177]: But even if you click on the right side it does not put you to the end of the text.

Comment (rating89us) [reply pages/History.qml]: ok, no problem

**Comment by selsta** [comment]
Please squash

**Review by selsta — APPROVED** [review]

**Pull request merged**